<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-06-15T18:06:33 IST" name="Suite" finished-at="2025-06-15T18:07:17 IST" duration-ms="43608">
    <groups>
      <group name="Master">
        <method signature="TC_001_AccountRegistrationTest.test_account_Registration()[pri:0, instance:testCases.TC_001_AccountRegistrationTest@655f7ea]" name="test_account_Registration" class="testCases.TC_001_AccountRegistrationTest"/>
        <method signature="TC_002_LoginTest.test_Login()[pri:0, instance:testCases.TC_002_LoginTest@39ad977d]" name="test_Login" class="testCases.TC_002_LoginTest"/>
      </group> <!-- Master -->
      <group name="Sanity">
        <method signature="TC_002_LoginTest.test_Login()[pri:0, instance:testCases.TC_002_LoginTest@39ad977d]" name="test_Login" class="testCases.TC_002_LoginTest"/>
      </group> <!-- Sanity -->
      <group name="Regression">
        <method signature="TC_001_AccountRegistrationTest.test_account_Registration()[pri:0, instance:testCases.TC_001_AccountRegistrationTest@655f7ea]" name="test_account_Registration" class="testCases.TC_001_AccountRegistrationTest"/>
      </group> <!-- Regression -->
    </groups>
    <test started-at="2025-06-15T18:06:33 IST" name="Test" finished-at="2025-06-15T18:07:17 IST" duration-ms="43608">
      <class name="testCases.TC_001_AccountRegistrationTest">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC_001_AccountRegistrationTest@655f7ea]" started-at="2025-06-15T18:06:34 IST" name="setup" finished-at="2025-06-15T18:06:46 IST" duration-ms="12013" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="test_account_Registration()[pri:0, instance:testCases.TC_001_AccountRegistrationTest@655f7ea]" started-at="2025-06-15T18:06:46 IST" name="test_account_Registration" finished-at="2025-06-15T18:06:59 IST" duration-ms="13257" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_account_Registration -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC_001_AccountRegistrationTest@655f7ea]" started-at="2025-06-15T18:06:59 IST" name="tearDown" finished-at="2025-06-15T18:07:00 IST" duration-ms="684" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC_001_AccountRegistrationTest -->
      <class name="testCases.TC_002_LoginTest">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC_002_LoginTest@39ad977d]" started-at="2025-06-15T18:07:00 IST" name="setup" finished-at="2025-06-15T18:07:04 IST" duration-ms="3575" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="test_Login()[pri:0, instance:testCases.TC_002_LoginTest@39ad977d]" started-at="2025-06-15T18:07:04 IST" name="test_Login" finished-at="2025-06-15T18:07:15 IST" duration-ms="11938" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1067)
at org.testng.Assert.assertEquals(Assert.java:1031)
at org.testng.Assert.assertEquals(Assert.java:1077)
at testCases.TC_002_LoginTest.test_Login(TC_002_LoginTest.java:38)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_Login -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC_002_LoginTest@39ad977d]" started-at="2025-06-15T18:07:16 IST" name="tearDown" finished-at="2025-06-15T18:07:17 IST" duration-ms="844" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC_002_LoginTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
